yt/visualization/volume_rendering/scene.py

```diff
@@ -437,15 +444,41 @@ def save_annotated(
         """
         fname = self._setup_save(fname, render)
 
-        # which transfer function?
-        rs = self._get_render_sources()[0]
-        tf = rs.transfer_function
-        label = rs.data_source.ds._get_field_info(rs.field).get_label()
-
         ax = self._show_mpl(
             self._last_render.swapaxes(0, 1), sigma_clip=sigma_clip, dpi=dpi
         )
-        self._annotate(ax.axes, tf, rs, label=label, label_fmt=label_fmt)
+
+        # number of transfer functions?
+        num_trans_func = 0
+        for rs in self._get_render_sources():
+            if hasattr(rs, "transfer_function"):
+                num_trans_func += 1
+
+        # which transfer function?
+        if num_trans_func == 1:
+            rs = self._get_render_sources()[0]
+            tf = rs.transfer_function
+            label = rs.data_source.ds._get_field_info(rs.field).get_label()
+            self._annotate(ax.axes, tf, rs, label=label, label_fmt=label_fmt)
+        else:
+            # set the origin and width and height of the colorbar region
+            if tf_rect is None:
+                tf_rect = 0.80, 0.12, 0.12, 0.9
```

Comment:
This is the easiest way to fix the newly introduced typing error. Alternatively you could just drop the type annotation for this variable as it's actually a little too restrictive (passing a tuple or np.ndarray would be equally fine, but it's currently hard to express this in a simple type annotation). Sorry about the churn I should have anticipated this...
```suggestion
                tf_rect = [0.80, 0.12, 0.12, 0.9]
```

Reply:
Not a big deal indeed but this actually makes it harder for downstream code to type check, FTR